docrepr/webview.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*
+
+"""
+View object documentation HTML in a separate GUI window.
+
+Requires PyQt4 or PySide. Original code by
+anatoly techtonik <techtonik@gmail.com> is placed in
+public domain.
```

Comment:
Please remove your name and different license from here. My reasons:
1. `docrepr` authors are also all Spyder authors (because this code is derived from it) and you're already one of them.
2. You have to accept the license of the project to contribute to it.